internal/flagdecorators/flagdecs.go

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package internal
```

Comment:
Do you think number of functions can be decreased by generalizing the functions? Like having `DecorateCommandWithStringArg` instead of separate `DecorateCommandWithJsonEntryFlag` and `DecorateCommandWithDelimiter`? Similarly for `time.Duration` related functions.

Reply:
> Do you think moving them to types/mapcmd is a good idea?

I guess, it is more better than keeping in internal.

> Do you think number of functions can be decreased by generalizing the functions?

Yes, I think we cannot generalize all of them but some of them can be generalized.

Reply:
`DecorateCommandWithDelimiter` has different default value than `DecorateCommandWithJsonEntryFlag`.